doc/content/reference/telemetry/identity_server.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Identity Server Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the IdentityServer"
+aliases: "/telemetry/identity-server"
+---
+
+{{% tts %}}'s Identity Server collects telemetry data, which is used to help us understand how users interact with the product. For example, the current data helps to have a better insight on the user's usage of organizations. Telemetry collection is **enabled** by default and does **not** collect personally identifiable information. The following section explains what information is collected and how it is anonymised.
+
+## User's Identification
+
+When the Identity Server starts the background task responsible for collecting and sending telemetry data, it generates the UID which is attached to every telemetry message that is sent. UID value is the SHA512 hash of `console.ui[IS,GS,NS,AS,JS].BaseURL` values.
+
+{{< note >}}
+For cases where using these values is not acceptable to the user, there is an option to set your own list of elements used to create the SHA512 hash. Edit [{{% tts %}}'s configuration]({{< ref "reference/configuration" >}}) and set `telemetry.uid-elements` to a list of string values of your choice.
```

Comment:
Replace `For cases where using these values is not acceptable to the user,` with `If you want the telemetry collector to use a different list of elements`.

Reply:
Changed a bit more than that, let me know what you think of the change